pandas/tseries/tools.py

```diff
@@ -224,6 +224,10 @@ def to_datetime(arg, errors='raise', dayfirst=False, yearfirst=False,
     infer_datetime_format : boolean, default False
         If no `format` is given, try to infer the format based on the first
         datetime string. Provides a large speed-up in many cases.
+    origin : scalar convertible to Timestamp / string 'julian', default None
```

Comment:
Sure. Would keep default as ``unix``.  A few of the cases for the use of arbitrary dates have been mentioned [here](https://stat.ethz.ch/R-manual/R-devel/library/base/html/as.Date.html).